ngraph/src/ngraph/op/proposal.cpp

```diff
@@ -54,12 +54,12 @@ void op::Proposal::validate_and_infer_types()
             class_probs_shape,
             ").");
 
-        NODE_VALIDATION_CHECK(
-            this,
-            class_logits_shape.size() == 4,
-            "Proposal layer shape class_logits_shape input must have rank 4 (class_logits_shape: ",
-            class_logits_shape,
-            ").");
+        NODE_VALIDATION_CHECK(this,
+                              class_bbox_deltas_shape.size() == 4,
+                              "Proposal layer shape class_bbox_deltas_shape input must have rank 4 "
```

Comment:
Please, update the  test file error message `ngraph/test/type_prop/proposal.cpp` in line 66

Reply:
Updated.